defaults/main.yml

```diff
@@ -38,6 +38,15 @@ xtradb_master_node: "{{ hostvars[ groups[xtradb_nodes_group][0] ].ansible_defaul
 # Defines the version
 xtradb_version: "57"
 
+# Defines the Debian release of the Percona repository
+# Please check your desired release [here](https://repo.percona.com/apt/)
+xtradb_repo_deb_release: "latest.generic"
```

Comment:
Are you sure this is OK?  Won't it generate:

  percona-release_latest.generic.buster_all.deb

... which is broken?  It should be just `latest`

Reply:
yup, you're right, it's a typo.